winch/filetests/filetests/x64/basic_add_with_locals.wat

```diff
@@ -0,0 +1,32 @@
+;;! target = "x86_64"
+
+(module
+  (export "main" (func $main))
+
+  (func $main (result i32)
+        (local $foo i32)  
+        (local $bar i32)
+        (i32.const 10)
+        (local.set $foo)
+        (i32.const 20)
+        (local.set $bar)
+
+        (local.get $foo)
+        (local.get $bar)
+        i32.add)
+)
+;;    0:	 55                   	push	rbp
+;;    1:	 4889e5               	mov	rbp, rsp
+;;    4:	 4883ec08             	sub	rsp, 8
```

Comment:
I just realized here that x64 also keeps `rsp` only 8-aligned -- this is fine for accesses to the stack in the function body (unlike aarch64) but will also need alignment when calls are implemented, because the x64 SysV ABI requires 16-alignment on call. (Function prologues can rely on this by using aligned SIMD stores...) Possible you're already thinking of this, but just wanted to note!